phpcs.xml

```diff
@@ -1,8 +1,30 @@
 ﻿<?xml version="1.0"?>
-<ruleset name="Laminas coding standard">
-    <rule ref="./vendor/laminas/laminas-coding-standard/ruleset.xml"/>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+    <arg name="basepath" value="."/>
+    <arg name="cache" value=".phpcs-cache"/>
+    <arg name="colors"/>
+    <arg name="extensions" value="php"/>
+    <arg name="parallel" value="80"/>
+
+    <!-- Show progress -->
+    <arg value="p"/>
+
+    <!-- set minimal required PHP version (7.3) -->
+    <config name="php_version" value="70400"/>
 
     <!-- Paths to check -->
     <file>src</file>
     <file>test</file>
+
+    <exclude-pattern>src/Int.php</exclude-pattern>
```

Comment:
Copied from `laminas-filter`? These classes don't exist here :thinking:

Reply:
e1cd856

Reply:
u were correct, ty for the fix